### PR TITLE
adjust navigation colors and small bugfixes

### LINKED ISSUE #14370: Regression issues regarding new project structure
# Bug Report

## Problem 
Some regression/unexpected behaviour after https://github.com/status-im/status-mobile/commit/c7d5e90882d709f15df32ec38e699f725ae546ae 

1) Black line (Android)
photo_2022-11-15 11 03 10 jpeg 2022-11-15 11-03-48

2) White lines / wrong background color for header/footer/ 'Jump to' shifted (Android)
photo_2022-11-15 11 16 27 jpeg 2022-11-15 11-17-00

photo_2022-11-15 11 18 10 jpeg 2022-11-15 11-18-42

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #14370

### CODE REVIEW

**Review by Parveshdhull — APPROVED** [review]

**Comment by churik** [comment]
After relaunch app (Dark mode on phone is on) got mixed dark/light modes in the app: 

https://user-images.githubusercontent.com/4557972/202749053-9437bbc3-b496-4425-97af-5b8a1dd9495e.mp4

Issue is not reproducible for nightly before project restructure, so it is regression

**Comment by flexsurfer** [comment]
@churik what do you mean by relaunch? just wondering why its only after relaunch

**Comment by churik** [comment]
I meant reopen the app

File: src/status_im2/navigation/roots.cljs [hunk 1]
```diff
@@ -5,23 +5,21 @@
...
-   :rightButtonColor     colors/neutral-100
-   :background           {:color colors/white}
-   ;; TODO adjust colors and icons with quo2
-   ;;:backButton
-   #_{:icon  (icons/icon-source :main-icons/arrow-left)
-      :color colors/neutral-100}})
+   :title                {:color (colors/theme-colors colors/neutral-100 colors/white)}
+   :rightButtonColor     (colors/theme-colors colors/neutral-100 colors/white)
+   :background           {:color (colors/theme-colors colors/white colors/neutral-100)}
+   :backButton           {:color (colors/theme-colors colors/neutral-100 colors/white)}})
```
Comment (ibrkhalil) [inline src/status_im2/navigation/roots.cljs]: DId Kebab casing not work for these props ?

**Review by ibrkhalil — APPROVED** [review]

**Comment by flexsurfer** [comment]
> I meant reopen the app

yeah but why, how is login different from reopening the app?  i mean there should be something, why its happens only when reopen?

**Comment by du82** [comment]
Bottom sheets create an off colored system bar

Screenshot_2022-11-19-22-12-14-13_65ad86f86fd2321a85272a99323da10f

**Comment by flexsurfer** [comment]
@du64 that's expected, new bottom sheet is under development

**Comment by flexsurfer** [comment]
@churik fixed

**Comment by churik** [comment]
@flexsurfer sorry for the delay in testing!

## ISSUE1: On Android currently, colors are mixed (dark theme only). 
Examples are below:

IMAGE 2022-11-22 10:02:47
IMAGE 2022-11-22 10:02:52
IMAGE 2022-11-22 10:03:07

## ISSUE1: On IOS white color flickering is noticeable on switching (dark theme)

https://user-images.githubusercontent.com/4557972/203272140-ca9201b5-06a7-4ae5-ba08-b76585c321c8.mp4


## ISSUE3: "Sign out" and "Jump to"  are overlapping on Android (Xiaomi Mi Note 9T) 

https://user-images.githubusercontent.com/4557972/203272960-656fc599-b812-4470-a8b4-88f4f5b868ff.mp4

## ISSUE4: On switching system theme app ends up in logged in state

Regression issue appeared after project restructuring.
### Steps:
- start multi-account creation
- on "Choose chat name"  put the app in the background
- change the system theme on your device (from dark > light or vice versa)


https://user-images.githubusercontent.com/4557972/203273553-e7acea55-88f3-4259-ac20-5cd999f7a8a6.mp4

**Comment by flexsurfer** [comment]
hey @churik thanks, that's all expected, we don't support old screens anymore

**Comment by flexsurfer** [comment]
so we should test only regressions in the new designs, and probably we could keep old screens at least usable, but they won't be perfect from design pov

**Review by ibrkhalil — APPROVED** [review]

**Comment by flexsurfer** [comment]
though I'll take a look at 2 and 4, thanks

**Comment by flexsurfer** [comment]
2 and 4 should be fixed now

**Comment by VolodLytvynenko** [comment]
Hi @flexsurfer. Thank you for your work. The current PR is tested and ready to be merged

**Pull request merged**